src/main/java/frc5124/robot2020/RobotMap.java

```diff
@@ -33,12 +32,13 @@
         public static final int shootLeaderCanID = 8; //8
         public static final int shootFollowerCanID = 6; //6
         public static final int shootSolenoid = 0;
-        public static final double Kp = 0.00021;
+        public static final double Kp = 0.000379; //,00021
         public static final double Ki = 0;
-        public static final double Kd = 0.00002;
-        public static final double Kf = 0.00008;
-        public static final double lineShootRPM = 4350; 
-        public static final double trenchShootRPM = 5200;
+        public static final double Kd = 0.005000;
+        public static final double Kf = 0.000040;
+        public static final double lineShootRPM = 4400; //4350
+        public static final double trenchShootRPM = 5300; //5200
```

Comment:
Why are we leaving the old values in constants? If it works better, can we just remove the old constants?

Reply:
I'd want to keep those old values in case of anything. It doesn't really hurt to have them. But, if you want to delete them, I'm okay with that too. My opinion on that is just kinda like we should try to do that so that if we have to change it, we have a baseline to change it from. But, these numbers aren't super different so, in this case I don't think it matters much.